pkg/account/component.go

```diff
@@ -75,6 +75,10 @@ func (c *component) UpdatePassword(ctx context.Context, currentPassword, newPass
 	}
 
 	_, err := c.keycloakAccountClient.UpdatePassword(accessToken, realm, currentPassword, newPassword, confirmPassword)
+	if err != nil {
+		c.logger.Warn("err", err.Error())
+		return err
```

Comment:
You should return nil in line 97

Reply:
But err should be nil in that case, no?

Reply:
Yes but writing `return err` let think that the returned value could be different

Reply:
True, you're right. I will change this